meson.build

```diff
@@ -58,6 +58,7 @@ services = [
     ['regulators', 'phosphor-regulators-config.service'],
     ['regulators', 'phosphor-regulators-monitor-enable.service'],
     ['regulators', 'phosphor-regulators-monitor-disable.service'],
+    ['regulators', 'reset_voltage_regulators.service'],
```

Comment:
Curious that this is using underscores instead of hyphens, but I do not see any sort of rule or coding convention that says you should not use underscores.

Reply:
Yeah, it would be ideal to keep this more consistent naming wise. But not a show stopper.

Reply:
Not a show stopper for me.

The only thing that seemed to reference which to use, that I found, was this: https://linuxhint.com/systemd_unit_file_service/
"So generally filenames in service only contains alphanumeric characters along with hyphens and underscores."

Nothing in the openbmc docs says we need one or the other, so I am good.